app/actions/NotificationActions.js

```diff
@@ -72,6 +79,15 @@ const transactionNtfnsDataHandler = (dispatch, getState) => {
 
       currentBlockHeight = null;
       currentBlockTimestamp = null;
+
+      if (accountMixerRunning) {
+        const changeAccount = sel.getChangeAccount(getState());
+        const { spendable } = await dispatch(getBalanceUpdateAttempt(changeAccount, 0));
+
+        if (spendable < MIN_RELAY_FEE_ATOMS) {
```

Comment:
I think a more appropriate value would be `(minimum_mix_denomation + min_relay_fee)`, but I'm not sure if dcrwallet or decrediton knows that value. Any ideas @jrick ?